packages/amplify-cli/src/index.ts

```diff
@@ -116,14 +119,61 @@ export async function run() {
     }
 
     return exitCode;
-  } catch (e) {
+  } catch (error) {
     // ToDo: add logging to the core, and log execution errors using the unified core logging.
-    errorHandler(e);
-    if (e.message) {
-      print.error(e.message);
-    }
-    if (e.stack) {
-      print.info(e.stack);
+    errorHandler(error);
+
+    if (error.name === 'JSONValidationError') {
+      const jsonError = <JSONValidationError>error;
+      let printSummary = false;
+
+      print.error(error.message);
+
+      if (jsonError.unknownFlags?.length > 0) {
+        print.error('');
+        print.error(
+          `The following feature flags are defined in configuration are unknown by the currently running version of Amplify CLI:`,
```

Comment:
These feature flags are defined in the amplify/cli.json configuration file and are unknown to the currently running Amplify CLI: